xivo/config_from_consul.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would prefer a separate repository for this tool and get leave lib-python as small as possible until it is removed